layouts/partials/style.html

```diff
@@ -323,8 +329,11 @@
     filter: invert(1) hue-rotate(180deg);
   }
 
-  body.dark-theme a.post-title,
-  .card-text {
+  body.dark-theme a.post-title {
```

Comment:
Can't these two be merged? Makes me think that there are others on this file, let's have a quick word before merging